Task8If/Task8If/Program.cs

```diff
@@ -0,0 +1,33 @@
+﻿using System;
+
+
+namespace Task8If
+{
+    internal class Program
+    {
+        static void Main()
+        {
+            Console.WriteLine("Вариант 21");
+            Console.WriteLine("Введите значение X");
+
+            double x  = double.Parse(Console.ReadLine());
+
+            Console.WriteLine("F(x) = " + calculation(x),3);
+
+
+            Console.ReadKey();
+        }
+
+        static double calculation(double x)
```

Comment:
1. Не вижу смысла во втором аргументе `Console.WriteLine()`. Он в данном случае ни на что не влияет.
2. Именуйте метод с большой буквы и, желательно, начинайте имя метода с глагола.
3. Не используйте лишних пустых строк.
```suggestion
            Console.WriteLine("F(x) = " + Calculate(x));

            Console.ReadKey();
        }

        static double Calculate(double x)
```